scripts/gofmt.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -eufo pipefail
+
+GIT_ROOT="$(git rev-parse --show-toplevel)"
```

Comment:
This won't work if not running from a git checkout.
Please determine folder the script is located at using
```bash
DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
```

Also the folder should not be changed, you could pass it to `find` instead:
```
ERR=$(find $DIR/.. -name *.go -exec go fmt "{}" \;)
```

Reply:
Updated.

> Also the folder should not be changed

It should be fine to change the folder, the script is launched in its own bash process including  a separate env.